Payments/Operation/05_ManageConfirmations.md

```diff
@@ -0,0 +1,123 @@
+[!!LIST (Confirmations](../UserInterface/06a_ListConfirmations.md)
+
+
+# Manage confirmations
+
+The *Confirmation* menu entry provides you with a view on all references to refunds that has been created. Before your company sends back the money to the customer, you have to check the refunds with this menu entry. This service is supported by the most payment service providers.
+> [Info] If a workflow decides that you have to sent back money to the customer, you have here the possibility to check the refunds before money is returned.   
+
+You can use this procedure to accept a refund so that the service payment provider gets the request to send back the money to the customer.    
+> [Info] The most payment service providers create a new reference, when a payment is to be refunded. However, the most payment service providers support an original reference ID that you can use to search for a specific case in the whole payment process.   
+
+You can also reject a refund, if it is not justified, for example, because the goods have already been delivered. In case of faulty transactions you can void them to clear the database.   
+
+In the settings might have been applied special permissions for this menu entry.
+> [Info] You might not be able to access this menu entry, because you have no permissions to do it. Furthermore, you may have access, but you are not permitted to execute refunds above a certain amount (for example, more than 50€). 
```

Comment:
```suggestion
> [Info] You might not be able to access this menu entry because you have no permissions to do it. Furthermore, you may have access, but you are not permitted to execute refunds above a certain amount (for example, more than 50 €). 
```

Reply:
done